src/main/java/io/conduit/SpecifierService.java

```diff
@@ -64,69 +69,61 @@ public void specify(Request request, StreamObserver<Response> responseObserver)
-                .setDefault("none")
-                .addValidations(requiredValidation)
-                .addValidations(availableConnectorsValidation(SinkConnector.class))
-                .build(),
-
-            "wrapper.schema",
-            Specifier.Parameter.newBuilder()
-                .setDescription("The schema of the records which will be written to a destination connector.")
-                .setDefault("none")
-                .build(),
-
-            "wrapper.schema.autogenerate.enabled",
-            Specifier.Parameter.newBuilder()
-                .setDescription(
-                    "Automatically generate schemas (destination connector). "
-                        + "Cannot be `true` if a schema is set.")
-                .setDefault("none")
-                .build(),
-
-            "wrapper.schema.autogenerate.name",
-            Specifier.Parameter.newBuilder()
-                .setDescription("Name of automatically generated schema. "
-                    + "Required if schema auto-generation is turned on.")
-                .setDefault("none")
-                .build(),
-
-            "wrapper.schema.autogenerate.overrides",
-            Specifier.Parameter.newBuilder()
-                .setDescription("A (partial) schema which overrides types in the auto-generated schema.")
-                .setDefault("none")
-                .build()
-        );
+    private Map<String, Specifier.Parameter> buildSourceParams() {
+        Map<String, Specifier.Parameter> params = new HashMap<>();
+        params.put("wrapper.connector.class", Specifier.Parameter.newBuilder()
+            .setDescription(
+                "The class of the requested connector. "
+                    + "It needs to be found on the classpath, i.e. in a JAR in the `libs` directory. ")
+            .setDefault("none")
+            .addValidations(requiredValidation)
+            .addValidations(availableConnectorsValidation(SourceConnector.class))
+            .build());
+        params.putAll(underlyingSourceParams());
+        return params;
     }
 
-    private Map<String, Specifier.Parameter> buildSourceParams() {
-        return Map.of(
-            "wrapper.connector.class",
-            Specifier.Parameter.newBuilder()
-                .setDescription(
-                    "The class of the requested connector. "
-                        + "It needs to be found on the classpath, i.e. in a JAR in the `libs` directory. ")
-                .setDefault("none")
-                .addValidations(requiredValidation)
-                .addValidations(availableConnectorsValidation(SourceConnector.class))
-                .build()
-        );
+    private Map<String, Specifier.Parameter> buildDestinationParams() {
+        Map<String, Specifier.Parameter> params = new HashMap<>();
+        params.put("wrapper.connector.class", Specifier.Parameter.newBuilder()
+            .setDescription(
+                "The class of the requested connector. "
```

Comment:
small thing but this description text does appear to be the same in both functions.